e2e-tests/cypress/support/appeal-statement-submission/confirmAppealStatementFileIsUploaded.js

```diff
@@ -1,7 +1,9 @@
 module.exports = (filename) => {
   cy.goToAppealStatementSubmission();
-  cy.get('#appeal-statement-file-name').invoke('text').then((text) => {
-    expect(text).to.eq(filename);
-  });
+  cy.get('#appeal-upload-file-name')
```

Comment:
these id's have been changed to match what is on the appeals service API. Previously they were set to whatever value was in the prototype.